src/Tracy/assets/BlueScreen/content.phtml

```diff
@@ -193,10 +193,11 @@ $code = $exception->getCode() ? ' #' . $exception->getCode() : '';
 		<?php $bottomPanels = [] ?>
 		<?php foreach ($this->renderPanels(NULL) as $panel): ?>
 		<?php if (!empty($panel->bottom)) { $bottomPanels[] = $panel; continue; } ?>
+		<?php $collapsed = isset($panel->collapsed) ? (bool) $panel->collapsed : true; ?>
```

Comment:
the `(bool)` cast is unnecessary; `true` should be written in uppercase (`TRUE`)